job/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -656,7 +657,7 @@ public int hashCode() {
             }
             node.addAction(new TimingAction());
 
-            logNodeMessage(node, "Running: " + node.getDisplayName());
+            logNodeMessage(node);
```

Comment:
Hm? Seems it deletes the "Running" text prefix. +1 from me for annotated logs, but I'm not sure how good it will appear in plain text